owtf/api/handlers/jwtauth.py

```diff
@@ -0,0 +1,64 @@
+# """
+#     JSON Web Token auth for Tornado
+# """
+
+import jwt
+secret_key = "my_secret_key"
```

Comment:
ditto - you have multiple places where this "key" is defined, it should be in a settings file

Reply:
Noted and changed.